### ISSUE TITLE
when output value is json or contains double-quotes, got wrong value in next step

### ISSUE THREAD

**Issue by lixiv** [yohamta/dagu#231]
examples:
```
steps:
  - name: "first"
    command: "echo {\"key\":\"value\"}"
    output: JSON
  - name: "second"
    command: "echo $JSON"
    depends:
      - first
```

**Comment by yohamta** [comment]
@lixiv I have tried to reproduce the issue, but it seems working correctly on my computer.
Could you please try updating to the latest version `v1.6.6` and check if the issue still happens?

![Dagu admin 2022-08-03 20-30-21](https://user-images.githubusercontent.com/1475839/182597662-8c068356-db7a-4027-9b92-30c393d6d898.jpg)

**Comment by lixiv** [comment]
> @lixiv I have tried to reproduce the issue, but it seems working correctly on my computer. Could you please try updating to the latest version `v1.6.6` and check if the issue still happens?
> 
> ![Dagu admin 2022-08-03 20-30-21](https://user-images.githubusercontent.com/1475839/182597662-8c068356-db7a-4027-9b92-30c393d6d898.jpg)

the expect result is {"key":"value"}, but got {key:value}

**Comment by yohamta** [comment]
Oh, sorry for my misunderstanding. I see. Will look at it.

**Comment by yohamta** [comment]
@lixiv Fixed in version `v1.6.7`. It was great that this issue has been discovered. Thank you very much!
